tests/playwright-test/assets/simple-server.js

```diff
@@ -11,4 +11,4 @@ setTimeout(() => {
       response.end(process.env.FOO);
     });
   });
-}, 750);
+}, +(process.argv[3] ?? '0'));
```

Comment:
```suggestion
}, process.argv[3] ? +process.argv[3] : 0);
```
this does not get transpiled so we have to use default Node.js syntax.